vite.config.ts

```diff
@@ -19,6 +19,8 @@ export default defineConfig({
       '@bcrs-shared-components/interfaces': new URL('./src/interfaces', import.meta.url).pathname,
       '@bcrs-shared-components/mixins': new URL('./src/mixins', import.meta.url).pathname,
       '@bcrs-shared-components/types': new URL('./src/types', import.meta.url).pathname,
+      '@bcrs-shared-components/services': new URL('./src/services', import.meta.url).pathname,
+      '@bcrs-shared-components/components/base-address': new URL('./src/components/base-address', import.meta.url).pathname,
```

Comment:
Why do you need this alias for base-address? Isn't it already handled by line 24?

Reply:
you are right. I realized this addition was redundant after I had pushed

Reply:
Please remove it in next commit.